src/index.js

```diff
@@ -1,29 +1,153 @@
+    if (e.key === 'Enter') {
+      const toDoContainers = document.querySelectorAll('.toDoContainer');
+      const localData = JSON.parse(localStorage.getItem('list'));
+      for (let i = 0; i < toDoContainers.length; i += 1) {
+        if (toDoContainers[i].classList.contains('checkedContainer')) {
+          localData[i].description = editInput.value;
+          localStorage.setItem('list', JSON.stringify(localData));
+        }
+      }
+      editInput.parentElement.classList.remove('checkedContainer');
+      toDoContainer.replaceChild(toDo, editInput);
+      toDo.textContent = editInput.value;
+    }
+  });
+  // remove or delete list
+  const removeIcons = document.querySelectorAll('.fa-trash');
+  removeIcons.forEach((i) => {
+    i.addEventListener('click', () => {
+      removeToDo(i.parentElement);
+    });
+  });
+};
 
-const listToDo = document.getElementById('listToDo');
+// general stracter for adding todo list
+const addTodo = (toDoValue) => {
+  const toDoContainer = document.createElement('div');
+  toDoContainer.className = 'toDoContainer';
+  toDoContainer.innerHTML += `
+<input type="checkbox" class="checkbox"/>
+<span>${toDoValue}</span>
+<i class="fa fa-ellipsis-v"></i>
+<i class="fa fa-trash"></i>
+`;
+  listToDo.appendChild(toDoContainer);
+  // the trash and 3 dots display block or none
+  const checkBox = document.querySelectorAll('.checkbox');
+  checkBox.forEach((i) => {
+    i.addEventListener('click', () => {
+      i.parentElement.classList.toggle('checkedContainer');
+      i.nextElementSibling.classList.toggle('checkToDo');
+      i.parentElement.lastElementChild.classList.toggle('trashActive');
+      i.parentElement.lastElementChild.previousElementSibling.classList.toggle('editDisable');
+      updateLocal();
+    });
+  });
+  // local storage
+  const object = new MyObject(toDoValue, false, checkBox.length);
+  myArray.push(object);
+  localStorage.setItem('list', JSON.stringify(myArray));
+  // edit listener
+  const editIcons = document.querySelectorAll('.fa-ellipsis-v');
+  editIcons.forEach((i) => {
+    i.addEventListener('click', () => {
+      editToDo(toDoContainer, i.previousElementSibling);
+    });
+  });
+};
+// edit (writing edit function)
+
+// adding to do list
+textInput.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter' && textInput.value) {
+    addTodo(textInput.value);
+    textInput.value = null;
+  }
+});
+/* eslint-disable  array-callback-return */
 
-const onLoad = () => {
-  listArray.forEach((task) => {
-    listToDo.innerHTML
-            += `<ul id="item-list" class="items">
-                        <li class="toDo">
-                            <button class="completed">□</button>
-                            <div class="discription">${task.description}</div>
-                            <div class="dots">⋮</div>
-                        </li>
-                    </ul>`;
+// showing data in the browser by bring it from local storage
```

Comment:
- [ ] Please, consider removing the liner disables from your code - let linter makes it a direct job - help you to find the bugs to fix them in the early stages of the development 👍 
- [ ] Also, I suggest you remove the comments, to make your code clear and professional. Your code is enough understandable and talks by itself about what is happening 👍